src/app/base/components/heuristic-statuses/heuristic-statuses.component.html

```diff
@@ -0,0 +1,28 @@
+<ng-container *ngIf="status !== 'CREATED' && status !== 'FINISHED' && status !== 'FAILED'"
```

Comment:
out of curiosity why are these with type check (=== vs ==)?

Reply:
No specific reason for here, to check the type and the value but in most cases we use the == operator, so maybe it would be better if we change it here so we can maintain consistency.